cdap-ui/app/cdap/components/PipelineNodeGraphs/PipelineNodeMetricsGraph.js

```diff
@@ -599,7 +612,11 @@ export default class PipelineNodeMetricsGraph extends Component {
             <div>
               <div className="title-container graph-title">
                 <div className="title"> {T.translate(`${PREFIX}.recordsOutTitle`)} </div>
-                <div className="total-records">
+                <div
+                  className="total-records"
+                  onMouseEnter={showPopoverOnHover ? this.togglePortsRecordsCountPopover : null}
+                  onMouseLeave={showPopoverOnHover ? this.togglePortsRecordsCountPopover: null}
+                >
```

Comment:
perhaps just do the check for showPopoverOnHover inside the function?

Reply:
Hmm I can't as that's not always the condition to call `this. togglePortsRecordsCountPopover()`... Have done some minor refactoring to avoid this repetition though